### PR TITLE
HiddenEye re-structurization

### LINKED ISSUE #359: Unexpected close after the first credentials capture
This error occurs after having first captured the user's credentials.

[*] Waiting For Victim Interaction. Keep Eyes On Requests Coming From Victim ...

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

..................................................................
[ CREDENTIALS FOUND ]:
[EMAIL]: 1223 [PASS]: 1234
..................................................................
Traceback (most recent call last):
File "./HiddenEye.py", line 38, in
getCredentials()
File "/root/Escritorio/HiddenEye/Defs/Actions.py", line 572, in getCredentials
log('..................................................................'.format(MAIN0, MAIN4))
NameError: name 'log' is not defined

Someone will know why this error occurs. (The script has root user permissions)

NOTE: Note that this error occurs in Localhost URL.

### PR DESCRIPTION
**Pull request by sTiKyt**
New structure, ready for 1.0
#400 
- Fixed lot of small issues
- Removed useless methods
- Fixed some typos
- Created Class-based system
- Removed "*" imports
- Fixed #359

### CODE REVIEW

File: Defs/Actions.py [hunk 1]
```diff
@@ -548,8 +551,7 @@ def mainMenu():
         system('clear')
         print('''
 
- {2} ██   ██ ██ ██████   ██████   ███████ ███   ██  {3}███████ ██    ██ ███████
- {1}
+ {2} ██   ██ ██ ██████   ██████   ███████ ███   ██  {3}███████ ██    ██ ███████{1}
  {2} ██   ██ ██ ██    ██ ██    ██ ██      ████  ██  {3}██       ██  ██  ██      {1}
  {2} ███████ ██ ██    ██ ██    ██ ███████ ██ ██ ██  {3}███████   ████   ███████ {1}
  {2} ██   ██ ██ ██    ██ ██    ██ ██      ██  ████  {3}██         ██    ██      {1}
```
Comment (sTiKyt) [inline Defs/Actions.py]: That was epic fail xD

File: Defs/Actions.py [hunk 2]
```diff
@@ -1,642 +1,767 @@
-#Any actions must be here
+# Any actions must be here
 
 from os import system, path
+from wget import download
+from platform import system as systemos, architecture
```
Comment (sTiKyt) [inline Defs/Actions.py]: @usama7628674 so, problem solved?

**Comment by usama7628674** [comment]
@sTiKyt No there a bug you can pull this PR and test it.Theres an error at port selection
```git pull origin pull/401/head```

**Comment by sTiKyt** [comment]
@usama7628674 somehow missed it when was debugging, fixed now.

**Comment by usama7628674** [comment]
@sTiKyt Thanks will test tomorrow.

**Comment by usama7628674** [comment]
@sTiKyt Can you fix the error that people get when they first run HiddenEye?
Its about Defaults something like that.

**Comment by sTiKyt** [comment]
@usama7628674 i think they have to reinstall HiddenEye or delete Settings.ini, that's the only way to fix it, this will be fixed in next pull request, when i'll be working on new backward compatible settings system. (maybe we can just delete Settings.ini automatically, but then this file will become useless :/)

I think i'll write "Fixing" script very soon. But not in this pull request.

**Comment by usama7628674** [comment]
@sTiKyt Okay sounds good

**Comment by sTiKyt** [comment]
> Please fix actions.py file.There are bugs.

**Comment by usama7628674** [comment]
@sTiKyt I'll test tomorrow and will review here and if everything goes right than you can merge it.

**Comment by sTiKyt** [comment]
@usama7628674 ok, waiting for it :D

**Comment by sTiKyt** [comment]
@usama7628674 so, what about this pull request?

**Comment by usama7628674** [comment]
@sTiKyt Well I tested it and there are still bugs.I cant get logs on terminal instead it dumps error.I've to fix it than after that i couldnt get victim public ip,current logged on user.

**Comment by sTiKyt** [comment]
@usama7628674 can you explain this problem better? With screenshots.

Seems like i have problems understanding what's wrong

**Comment by Moltivie** [comment]
@sTiKyt Hey, what you came up with on this one?

May I get a TODO on this branch, or just directly asking you what should be have to be done here.
Thanks.

**Comment by sTiKyt** [comment]
@Moltivie don't have enough time to find and fix all bugs :/
And it won't be accepted until someone do..

**Comment by Moltivie** [comment]
Accepted what? 🤔

**Comment by sTiKyt** [comment]
@Moltivie branch... Branch won't be merged until it has bugs

**Comment by Moltivie** [comment]
Yeah, that's what I was asking for. If you could note me what you are working on and a TODO list, we can figure this out both.

**Comment by sTiKyt** [comment]
@Moltivie trying to find bugs @usama7628674 talking about.. Read comments above

**Comment by Moltivie** [comment]
@sTiKyt I am going to test it out.

**Comment by Moltivie** [comment]
@sTiKyt how you doing with this one?
You may want to change some codes, as the previous you were working one, I pretty much prettify it a bit.

**Comment by sTiKyt** [comment]
@Moltivie Now i don't have time to work on it :/
Maybe you can help to solve this PR

**Comment by sTiKyt** [comment]
Too much needs to be redone, closing here...

**Pull request closed without merging**